.openshift/Pipfile

```diff
@@ -15,12 +15,10 @@ geffnet = "==0.9.8"
 fastseg = "*"
 onnx = "*"
 ipywidgets = "*"
-torch = "==1.5.1"
-torchvision = "==0.6.1"
+torch = "<=1.7.1+cpu,>=1.5.1+cpu"
```

Comment:
Should we pin PyTorch to a specific version here, since it will be pinned to a specific version in the Pipfile.lock file anyway?

Reply:
Good idea. Updating it

Reply:
I could not get this to work by changing the Pipfile and requirements.in. Let's leave it like this for now. I want to address the security issues first.